README.md

```diff
@@ -1,82 +1,68 @@
-django_cinema/README.md¶
-
-                               =============
-                               Django Cinema
-                               =============
-
+README.md
+***
+#DjangoCinema
 Website of Cinema Library written in Django.
+***
+####Include:
 
-Include:
-    - Python
-    - Django
-    - PostgreSQL
-    - Celery
-    - Redis
-    - Django Rest Framework
-    - Gunicorn
-    - Nginx
-    - Docker
-
-Database diagrams:
-    - db_diagram/short_diagram.svg
-    - db_diagram/full_diagram.svg
-
+- Python
+- Django
+- PostgreSQL
+- Celery
+- Redis
+- Django Rest Framework
+- Gunicorn
+- Nginx
+- Docker
+***
+####Database diagrams:
 
-                               -----------
-                               Quick start
-                               -----------
-
-                               DEVELOPMENT
-
-To run current project locally you need:
+- db_diagram/short_diagram.svg
+- db_diagram/full_diagram.svg
+***
+#Quick start
+###I. Development
+####To run current project locally you need:
 
 1. Requirements:
-   - '.env.dev' file with environment variables;
-   - '.env.dev.db' file with DB environment variables;
-2. Start by cloning down this project from GitHub repository.
-3. Set up Python development environment.
-4. Install Docker and Docker-compose.
+    - '.env.dev' file with environment variables
+    - '.env.dev.db' file with DB environment variables
+2. Start by cloning down this project from ***GitHub*** repository.
+3. Set up ***Python*** development environment.
+4. Install ***Docker*** / ***Docker-compose***.
 5. Run 'docker-compose up -d --build'.
 6. Run 'docker exec -it django_web python manage.py flush --no-input'.
 7. Run 'docker exec -it django_web python manage.py migrate'.
-8. Run 'docker exec -it django_web python manage.py shell
-   --command="from django.contrib.contenttypes.models import ContentType;
-   ContentType.objects.all().delete(); exit()"'.
-9. Run 'docker exec -it django_web python manage.py loaddata
-   django_cinema/fixtures/db.json'.
-10. Open browser at 'http://localhost:8000/' to visit website.
-
-                               PRODUCTION
+8. Run 'docker exec -it django_web python manage.py shell --command="from django.contrib.contenttypes.models import ContentType; ContentType.objects.all().delete(); exit()"'.
```

Comment:
If you want to highlight that this is a command, that is good, please use the appropriate syntaxis for it.
For instance,

Run `docker exec -it django_web python manage.py shell --command="from django.contrib.contenttypes.models import ContentType; ContentType.objects.all().delete(); exit()"`

Reply:
ok